File: 8_Ball_Project/main.cs
using System;
using System.IO;

namespace eight_Ball
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Welocome to 8 Ball\nWrite your question here: ")
            string user_Input = Console.Readline();
            Random rnd = new Random();
            int eight_Ball_Rnd_Number = rnd.Next(1, 21);
            Console.WriteLine("This is a random number " + eight_Ball_Number);   
        }
    }
}

File: 8_Ball_Project/main.cpp
#include <iostream>
#include <string>
#include <fstream>

using namespace std;

int main() {
  srand(time(0));
  int randomNum = rand() % 20 + 1;
  string user_Input;
  


  cout << "\nWelcome to 8Ball!\n\nPlease write your question here: ";
  cin >> user_Input;
  
  

  return 0;
} 